src/Store/VimStoreConnector.re

```diff
@@ -842,6 +842,14 @@ let start =
       ();
     });
 
+  let saveEffect =
+    Isolinear.Effect.create(~name="vim.save", () => {
+      let _ = Vim.input("<esc>");
+      let _ = Vim.input("<esc>");
+      let cursors = Vim.input("w");
+      ();
+    });
```

Comment:
`w` in normal mode will just move the cursor. What we want is to emulate `:w` followed by enter. Something like this should work I think:

```suggestion
  let saveEffect =
    Isolinear.Effect.create(~name="vim.save", () => {
      let _ = Vim.input("<esc>");
      let _ = Vim.input("<esc>");
      let _ = Vim.input(":");
      let _ = Vim.input("w");
      let _ = Vim.input("<CR>");
      ();
    });
```

Reply:
@glennsl Good catch! I just updated the PR.